### ISSUE TITLE
Screen readers do not read the first selection of the DropDownList

### ISSUE THREAD

**Issue by kdikov82** [telerik/kendo-react#1392]
Opening the popup and clicking on item for the first time does not announce the selected value. Works correctly for next selections:

- [https://stackblitz.com/edit/react-nuw481?file=app%2Fmain.jsx](https://stackblitz.com/edit/react-nuw481?file=app%2Fmain.jsx)

Tested with NVDA and KendoReact 5.5.0

**Comment by kdikov82** [comment]
Additionally, clicking to open the menu, and then you using the arrow keys to move go up or down through the item list, none of the items are read aloud.

**Comment by kdikov82** [comment]
Tested with version 5.11.0 and both issues are resolved:
https://stackblitz.com/edit/react-nuw481-o8b9pv?file=app%2Fmain.jsx
